### ISSUE TITLE
As an admin, I want to create or update a Zotero library with data for all works referenced in a Derrida work so that I can make the data available for others to use.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/derrida-django#230]
## Notes for development
- Custom a manage.py script to populate a Zotero library using the Zotero API and pyzotero library.
- Write a method to export a Work Instance as a Zotero item; may want to reference the previous project private Zotero library to compare types and other fields.
- Should ideally support create as well as update so that changes in the project database can automatically be propagated to the Zotero library. (We'll presumably need unique, stable ids for this).
- Zotero library or id should be configured in local settings; one library per Derrida work in the system (only _de la grammatalogie_ for now)
- If it's easy, the script could also include an option to generate one or more Zotero exports to be deposited as a versioned dataset (CSV, CSL JSON, perhaps also an RDF).

**Comment by rlskoeser** [comment]
Added new zotero configurations to the deploy, deployed to QA, ran the export_zotero script on a private group library for testing.  Script output:
```
Found 1 new Derrida work.
Exporting 204 instances.
100% (204 of 204) |#############################################################################################| Elapsed Time: 0:03:14 Time:  0:03:14

Export complete.
Created 204; updated 204; unchanged 0; failed 0.
```

**Comment by thatbudakguy** [comment]
@rlskoeser did you end up figuring out why it was "updating" about half and leaving the rest unchanged each time? just a fluke on my local instance?

**Comment by rlskoeser** [comment]
@thatbudakguy not just a fluke on your end, I was seeing it too. I wanted to investigate more, but needed to move on. I think it's not a big deal for the moment, but would be interested to revisit the next time we work on it and see if we can determine what's going on. I think we could also clean up reporting a little bit better, e.g. the updated total currently includes new items and we could pretty easily subtract those.

The script was surprisingly slow when I ran it in QA, I'd also be interested in looking into that and seeing if there's anything we can do to improve the speed. Even if it's all on the Zotero end, maybe smaller chunks would be faster? We could ask on the forums.

It would be nice to have you to review the changes I made when you're back next wek.
